.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     'prettier/standard',
   ],
   rules: {
-    'prettier/prettier': ['error', 'fb'],
```

Comment:
Why is this needed?

Reply:
`eslint-plugin-prettier` dropped support for it. ([source](https://github.com/prettier/eslint-plugin-prettier/commit/478c7e5d2165f3e67e893c9a317b602159eaff9c))
> Drop support for specifying prettier options using the "fb" shorthand
string. Replace "fb" with either config in your .prettierrc or the object:
`{ singleQuote: true, trailingComma: 'all', bracketSpacing: false, jsxBracketSameLine: true, parser: 'flow'}`